### PR TITLE
MS - Skip test cases which are covered in the new MS specific acceptance test

### PR DESCRIPTION
**Pull request by jilju**
Skip the test cases which are covered in the [new acceptance test](https://github.com/red-hat-storage/ocs-ci/pull/5588) created for managed services.

Signed-off-by: Jilju Joy <jijoy@redhat.com>

### CODE REVIEW

**Comment by jilju** [comment]
New MS acceptance test is added in https://github.com/red-hat-storage/ocs-ci/pull/5588.
This PR can be merged after merging 5588

**Comment by jilju** [comment]
Verified:
```
$ run-ci  --cluster-path /home/jijoy/ms-provider/ --ocs-version 4.10 --ocp-version 4.9 --ocsci-conf conf/deployment/rosa/managed_3az_consumer_qe_3m_3w_m54x.yaml tests/manage/pv_services/pvc_clone/test_pvc_to_pvc_clone.py tests/manage/pv_services/pvc_snapshot/test_pvc_snapshot.py tests/manage/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py tests/manage/pv_services/test_raw_block_pv.py  --dev-mode --cluster-name sgatfane-30pr --dev-mode 
2022-03-30 16:15:45,411 - MainThread - ocs_ci.framework.pytest_customization.ocscilib - INFO - Pytest configure switching to: cluster=0
2022-03-30 16:15:46,589 - MainThread - ocs_ci.framework.pytest_customization.ocscilib - INFO - Dump of the consolidated config file is located here: /tmp/run-1648637141-cl0-config.yaml
2022-03-30 16:15:46,589 - MainThread - ocs_ci.framework.pytest_customization.ocscilib - INFO - Running in development mode
======================================================================================== test session starts =========================================================================================
platform linux -- Python 3.9.10, pytest-6.2.5, py-1.10.0, pluggy-0.13.1
rootdir: /home/jijoy/ocs-ci, configfile: pytest.ini
plugins: flaky-3.7.0, repeat-0.9.1, html-2.1.1, metadata-1.11.0, logger-0.5.1, ordering-0.6, marker-bugzilla-0.9.4, reportportal-1.10.3
collecting ... 
---------------------------------------------------------------------------------------- live log collection -----------------------------------------------------------------------------------------
16:15:46 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc get Node  --selector=node-role.kubernetes.io/worker -o yaml
16:15:48 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc get Node  --selector=node-role.kubernetes.io/infra='' -o yaml
collecting 12 items                                                                                                                                                                                  16:15:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc get clusterversion -n openshift-storage -o yaml
16:15:52 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc get clusterversion -n openshift-storage -o yaml
16:15:53 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc get clusterversion -n openshift-storage -o yaml
16:15:54 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc get clusterversion -n openshift-storage -o yaml
collected 12 items                                                                                                                                                                                   

tests/manage/pv_services/pvc_clone/test_pvc_to_pvc_clone.py::TestClone::test_pvc_to_pvc_clone[CephBlockPool] SKIPPED (Test will not run on Managed service cluster)
tests/manage/pv_services/pvc_clone/test_pvc_to_pvc_clone.py::TestClone::test_pvc_to_pvc_clone[CephFileSystem] SKIPPED (Test will not run on Managed service cluster)
tests/manage/pv_services/pvc_snapshot/test_pvc_snapshot.py::TestPvcSnapshot::test_pvc_snapshot[CephBlockPool] SKIPPED (Test will not run on Managed service cluster)
tests/manage/pv_services/pvc_snapshot/test_pvc_snapshot.py::TestPvcSnapshot::test_pvc_snapshot[CephFileSystem] SKIPPED (Test will not run on Managed service cluster)
tests/manage/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py::TestDynamicPvc::test_rwo_dynamic_pvc[CephBlockPool-Retain] SKIPPED (Test will not run on Managed service c...)
tests/manage/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py::TestDynamicPvc::test_rwo_dynamic_pvc[CephBlockPool-Delete] SKIPPED (Test will not run on Managed service c...)
tests/manage/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py::TestDynamicPvc::test_rwo_dynamic_pvc[CephFileSystem-Retain] SKIPPED (Test will not run on Managed service ...)
tests/manage/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py::TestDynamicPvc::test_rwo_dynamic_pvc[CephFileSystem-Delete] SKIPPED (Test will not run on Managed service ...)
tests/manage/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py::TestDynamicPvc::test_rwx_dynamic_pvc[CephFileSystem-Retain] SKIPPED (Test will not run on Managed service ...)
tests/manage/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py::TestDynamicPvc::test_rwx_dynamic_pvc[CephFileSystem-Delete] SKIPPED (Test will not run on Managed service ...)
tests/manage/pv_services/test_raw_block_pv.py::TestRawBlockPV::test_raw_block_pv[Delete] SKIPPED (Test will not run on Managed service cluster)
tests/manage/pv_services/test_raw_block_pv.py::TestRawBlockPV::test_raw_block_pv[Retain] SKIPPED (Test will not run on Managed service cluster)

======================================================================================== 12 skipped in 9.03s =========================================================================================

```

**Review by sidhant-agrawal — APPROVED** [review]

**Pull request closed without merging**
